src/GitVersion.Core/VersionCalculation/VersionSearchStrategies/IBaseVersionIncrement.cs

```diff
@@ -6,5 +6,7 @@ public interface IBaseVersionIncrement
 {
     string Source { get; }
 
+    VersionIncrementSourceType SourceType { get; }
```

Comment:
Can you please explain me for what the enum `VersionIncrementSourceType` is good for? In my opinion the hole SourceType property can be removed.